src/CacheControl/CacheControl.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Nuwave\Lighthouse\CacheControl;
+
+class CacheControl
+{
+    /**
+     * List of maxAges.
+     *
+     * @var array<int, int>
+     */
+    protected $maxAgeList = [];
+
+    /**
+     * List of scopes.
+     *
+     * @var array<int, string>
+     */
+    protected $scopeList = [];
+
+    public function addToMaxAgeList(int $maxAge): void
+    {
+        $this->maxAgeList[] = $maxAge;
+    }
+
+    public function addToScopeList(string $scope): void
+    {
+        $this->scopeList[] = strtolower($scope);
```

Comment:
```suggestion
        $this->scopeList[] = $scope;
```

Let's just always use `PUBLIC` and `PRIVATE`.